src/test/java/com/softserveinc/ita/dkrutenko/RozetkaTests.java

```diff
@@ -23,14 +25,13 @@ public void verifySearchTest(String brandNameItem, String requiredItem) {
         //take list of items and filter our required item. Check if that item are actually present
         var actualItem = searchGoods.getRequiredProductName(requiredItem);
         Assert.assertTrue(actualItem.contains(requiredItem));
+        searchGoods.clickLogoIcon();
```

Comment:
probably after changing from @BeforeClass to @BeforeMethod this click will not need to be

Reply:
removed line #28